mlrun/db/httpdb.py

```diff
@@ -239,16 +248,22 @@ def api_call(
 
         return response
 
-    def _init_session(self):
+    def _init_session(self, retry_on_post: bool = False):
         return mlrun.utils.HTTPSessionWithRetry(
             retry_on_exception=config.httpdb.retry_api_call_on_exception
-            == mlrun.common.schemas.HTTPSessionRetryMode.enabled.value
+            == mlrun.common.schemas.HTTPSessionRetryMode.enabled.value,
+            retry_on_post=retry_on_post,
         )
 
     def _path_of(self, prefix, project, uid):
         project = project or config.default_project
         return f"{prefix}/{project}/{uid}"
 
+    def _is_retry_on_post_allowed(self, method, path: str):
+        return method == "POST" and any(
```

Comment:
if you are here then method is definitely POST, don't think you need to check again

Reply:
added a comment